java/arcs/core/data/expression/Expression.kt

```diff
@@ -209,7 +209,7 @@ sealed class Expression<out T> {
         /** Boolean negation. */
         object Not : UnaryOp<Boolean, Boolean>() {
             override operator fun invoke(expression: Boolean): Boolean = !expression
-            override val token = "!"
+            override val token = "not"
```

Comment:
To be honest, it might be cleaner to use the '&&', '||', '!' operators for these, but I guess lets standardize and then improve later.

Reply:
I took these from the RefinementExpressions in pegjs, is there a reason why the refinement's dont use &&/||?

Reply:
We were attempting to match the check + claim syntax (yes, a 3rd sub syntax in the Arcs language).
They attempt to use natural language rather than C/Python style.

I think paxel is reason enough to diverge from that approach, and maybe we can convince @csilvestrini  + @bgogul to switch over?